lib/runner.ts

```diff
@@ -272,26 +272,28 @@ export class Runner extends EventEmitter {
       browser_.ng12Hybrid = initProperties.ng12Hybrid;
     }
 
-    browser_.ready =
-        browser_.ready
-            .then(() => {
-              return browser_.waitForAngularEnabled(initProperties.waitForAngularEnabled);
-            })
-            .then(() => {
-              return driver.manage().timeouts().setScriptTimeout(
-                  initProperties.allScriptsTimeout || 0);
-            })
-            .then(() => {
-              return browser_;
-            });
+    browser_.ready = browser_.ready
+                         .then(() => {
+                           return initProperties.waitForAngularEnabled;
+                         })
+                         .then((waitForAngularEnabled) => {
+                           return browser_.waitForAngularEnabled(waitForAngularEnabled);
```

Comment:
Yup, I agree. waitForAngularEnabled is a promise and the promise is not resolved in the waitForAngularEnabled set method. So this makes sense.